examples/npc_only_with_traffic_lights_REST.py

```diff
@@ -0,0 +1,66 @@
+from PIL import Image as PImage
```

Comment:
Add something to the README to inform people what this example does.

Reply:
Added the following:
## Controlling the traffic-lights
Inverted AI API allows for controlling the traffic light states. 
We provided a [sample code](https://colab.research.google.com/github/inverted-ai/invertedai-drive/blob/develop/examples/npc_only_with_traffic_lights_REST.py)
for doing so at a traffic-light controlled intersection.